test/gaudi_opts/test_global_converter_maps.py

```diff
@@ -67,8 +67,8 @@
     podioOutput = PodioOutput("OutputWriter")
     podioOutput.filename = "global_converter_maps.root"
 
-PseudoRecoAlg = PseudoRecoAlgorithm(
-    "PseudoRecoAlgorithm", InputMCs=["MCParticles"], OutputRecos=["PseudoRecoParticles"]
+PseudoRecoAlg = PseudoRecoFunctional(
+    "PseudoRecoFunctional", InputMCs=["MCParticles"], OutputRecos=["PseudoRecoParticles"]
```

Comment:
Can we also make this a "two-pronged" test? I.e. run it once with `Functional` and once with `Gaudi::Algorithm`? Otherwise we might run into similar issues as we are fixing here again at some point.